sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -74,6 +74,7 @@ export interface PollOperation<TState, TResult> {
 // @public
 export interface PollOperationState<TResult> {
     error?: Error;
+    initialRawResponse?: unknown;
```

Comment:
Are there any common properties of the raw response that we can use to provide a more descriptive type? I'm not opposed to keeping the `unknown` type, just wondering if there is any additional help we can provide